cmd/generate/generate.go

```diff
@@ -50,6 +52,12 @@ func NewGenerateCommand(cfg *command.Config) *cobra.Command {
 				return fmt.Errorf("failed to parse flags: %w", err)
 			}
 
+			// Parse template variables from flags
+			templateVars, err := util.ParseTemplateVariables(cmd.Flags())
+			if err != nil {
```

Comment:
Ensure that "input" is not a template bars key

Reply:
input is allowed for "run" command but not for "generate" command. Move the check to the generate command code.